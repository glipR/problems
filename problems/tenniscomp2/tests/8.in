256
mawanv0v7kh7umxvw4l5
rwph4x6x
tlq8xe4vbhl
v7sshx7yvkzrfg3fwq
6gawp4fgj
h9fv3wwz
0ros9pgu96u4rthan
kmhzmpg25qe3ijkqj2b
3qgpj9l
1b4ohj7tc2pw5at
yxcaxxvw76ku8cngqrv
2lacdcq
u925v
8hbohq
7lvts2dckldnp1i4a
1q75y29ubv
not9618gq2h3sioltx
ubjvrex91
tv9kgh
e4hk5knjp9uw
ewyoy421ainvrb
hplsgkx21x6aer0
ue7hydhf
537oz9dmht
fkfrv2gf9cys
yzgfwvtwfn4z2e9
h38a9oq65g17w
fo1wdzs0gyss
ya8fx953krdpnqho5fm
66fv861vgv5zy5x16y
8nxtj
zsgy3696xltij50590o
6ynb1wzsztreni
n8f4w
5402e60arjbei
4nrzgy
yo5qq
vstin1fttrh1hm6d0
93a27r
grlf9oq81y2fb8dqij
tgquxqmqkxb3v2ozvir
ifm93ajsoboyhi7
7gm456iuv
8ltd2ateva7
rlb8pcl6c
756swkupij2jard1zgb
9v1u7zr2k31wp7
rhy4xc3yqpxk9
rv50yzt4wzx
rhzbx5amli4os8jkvpa0
dgns5kx077xe
lpbv8
6p5cq6qd9n9
tldj0beb6qgg
kpl0ioo
8q6i9
q4koswzeos6
3gktd1lg6w9
nxxclqusf
xe6od1fdd1q
py03tycrmfnfw9wn27f
dhtl7b3v
rmf2dx5
ehvz4
jxihwh
h0nirf80mth62qz
yfx7evxtec
2lg5dpbv1n0jq
urpzr4w70q
mp39vqtv
16s13
r07oy2og5tsqzabcj1
1g15ditjqt33j7luxuy
91lqbkl2bv49rh3
rfhg1xh2ftjwsku830i
mm0j0t5m8h9rywxf62s
vemk9pho
bmzdrb
c544p65vf
khj3jmiz29bo6bbmom
ew4iudjxpe4dan96ewp
june4njm
dsy8q85
z6v5938
y11bu2md
mq8qxfaot
s2d2lp43q2a8pz6n6
wi9vx7jsopu6izpx
wuvr5r12tlqic8j0dz
mrr5f6zrqydf1ouhxblg
eobk4vubnrw1d
vsaijk
2rdp0qzg0m34utkqdf
lzmv4rh7tu
n7nqrsg5n
swzkx5a181pg
wu24pf
5ap1355sbeqs4cq70
ofiqnwzgdflpji1q8cm
0sutt99i0g3rx7vd3g
mjbtj3
jylbfl3ejdxu6j
kvdr0cwbkwxpkgyha
yjgm3iosytcg
m368u7k6znjy
omlnjw5li
o0lzce1go2txc
4rtf7w996y2to8zjfj
elxofq9txd9yn6l
zv07i
o6gfx
3cww0ampgn
v67i7it0j
66hbk3nsjj704ej
5ti4epdo9j0z6i2a
xyoo6sd8v4po
ygbkspt127ecdvz9odm6
i6azkzbyyrr
5fcufl17rc5j
fvez44gv
ag91h859pkhc2
ewuouj9vof918
d5df1jt0
l3tmxc4g7r9g
qzcl6
7zuwwul
gp82s7
w4bbuqrjuzt7yrh
o0sy724gex1x39
emc2qs
gdr3p9x2iqeoyucd
w0u9f6fassx7wf1s7bf
4v0zm1yvtyr21hn
lle22z0je59
lj0jy
dsvowah1dcqmd0
57re9da3f46
qugtnbrmxnktys9y
ahq79
h0epewa
lo3mqa5
jtgpg8d
jr2zesam6u
o8blu
7v29i4a29va9
400yzlummsdqjebsf7q6
btqwzb2
rmt1m5w5nxe
wj5dfa05mu8k7uiti
ybuh00emauudh9
fe65wvwafy
ezf517p04epn
lbua1diiwubh69vcz
kj089cg
4fqv84orwo27j1zxmi75
1ap1wr2fso7ule0edf9
8s1yuvdr9t1
2bzu0jr2
2gr2u2ua4lwpz4uoxkzl
p1u2ji9aa7ttk
49vox4g7l0mgwhle4sxc
yasu9
7oyjegwp1
jcjw5i6
01ncfey5a35
49u0l95q7dk9gnahcx
jmahlsvsun84b8ng1z0
7vkkifbgbolv2
z3tz5ttjqpojy2o32er4
isd1ybfz948e
2ox5xu
rezryww
lmjtmw9mpznuge3mdf
5bmgetqe5jdzp2
hbj3e7
h19dh3yksto5niam
ied5bphnjyv8kmonac
wnrl5yufh5issdygkn7t
p7izj8c0xwlnrc5r
x5r783lzq5
llwvcf3dfxn14i9ogypk
as2n83d9bq792hgnbifs
bl71h1vd8
0s7rd4fvn1u5fuhcq91x
hlqcd
p6bp35vpug3aztcyaolj
kqn3i2kcli6p4hr4s4
56le3fn
v4capsoqzzhgbex
8rkapv7c6e3iebt0xny
hbcubmq1j2yl
432kx3y
n2sx4tz
jwmuculzh214bnzymth
keciawvlv8zngf30m24
h89m5z
c6yaekn4u9yq5z4epgje
idwjrv0o7k03
t2cbq3l5rs53d2y8
oaxz4kbpqgex9p0izms
deuyxioz
jef6w
fyeq61k0x10iu5vf6rv
s0cyoijjjfnea
mopgbtfh
gvuba08f032f26fc08b
38fme3f6jfl
g6owktf3mm
wyc73
xq7r2qs15x4eiz8uh3
s67joxsfz7r98rnwhes
a6ngwvkail7rzixr4
fpvgacdyz1ei64qk55u0
6jsoygtkhxw
xjow9ve
r0od3mhl7ds
n1av6q1xi2mahhs
cdrqcl0gjejm8bvtyi
mx1e2rp8
8h55r
2wqvcuzfgvnz3r9jtu9o
2omf4i
7gv40vhzzw9zj
edfuexo
ihq1ynn7e8wjv
2udxaa
bzuz1j16yh38pi904zqd
vyqkq3hm1
8k5izu0lg
lk849d6or
zafdqh7u
itpmz6bi
bcv169nfvb0xwbdg4g31
t8ks19jep6oap2lr1gbe
lbmu1ndivah9jt
o2e75evgz
brrrbqk7v0w7nuwm
prtrs3v
dhwyne6s1tigee02kw89
zgexeg59m
e2o5riz6jxpy
yw9vjw7ck
afhj383bvypkxw9p
2j25of2krsyzv0j3niy
ef30jkl3ie36xx84
cqo2h59v9nnci
i6wwimj37kpvlnlq
at9le2
uri9lq
wkre5hw89r2fmzyt
2vvimv07hyukhzwe
7oob8gb
pbx0129ajv5iq1ca
z0ann9j2f1io5
nk5d66fgj9p
k57oqje
2lg5dpbv1n0jq defeats urpzr4w70q
o0lzce1go2txc defeats 4rtf7w996y2to8zjfj
e4hk5knjp9uw defeats ewyoy421ainvrb
rwph4x6x defeats mawanv0v7kh7umxvw4l5
rlb8pcl6c defeats yo5qq
jef6w defeats deuyxioz
o6gfx defeats elxofq9txd9yn6l
tldj0beb6qgg defeats 6p5cq6qd9n9
fvez44gv defeats 5fcufl17rc5j
ag91h859pkhc2 defeats w4bbuqrjuzt7yrh
5402e60arjbei defeats n8f4w
k57oqje defeats nk5d66fgj9p
5bmgetqe5jdzp2 defeats lmjtmw9mpznuge3mdf
gdr3p9x2iqeoyucd defeats dsvowah1dcqmd0
rwph4x6x defeats tlq8xe4vbhl
yo5qq defeats 93a27r
zgexeg59m defeats dhwyne6s1tigee02kw89
eobk4vubnrw1d defeats mrr5f6zrqydf1ouhxblg
xyoo6sd8v4po defeats 66hbk3nsjj704ej
z0ann9j2f1io5 defeats wkre5hw89r2fmzyt
ue7hydhf defeats 537oz9dmht
ewyoy421ainvrb defeats ue7hydhf
qugtnbrmxnktys9y defeats h0epewa
qugtnbrmxnktys9y defeats gdr3p9x2iqeoyucd
w4bbuqrjuzt7yrh defeats qzcl6
june4njm defeats ihq1ynn7e8wjv
i6azkzbyyrr defeats fvez44gv
jmahlsvsun84b8ng1z0 defeats 7vkkifbgbolv2
2bzu0jr2 defeats 4fqv84orwo27j1zxmi75
qugtnbrmxnktys9y defeats 57re9da3f46
ifm93ajsoboyhi7 defeats tgquxqmqkxb3v2ozvir
49vox4g7l0mgwhle4sxc defeats yasu9
3qgpj9l defeats 2lacdcq
93a27r defeats grlf9oq81y2fb8dqij
rhy4xc3yqpxk9 defeats 9v1u7zr2k31wp7
8ltd2ateva7 defeats ifm93ajsoboyhi7
jmahlsvsun84b8ng1z0 defeats 2bzu0jr2
r0od3mhl7ds defeats n1av6q1xi2mahhs
400yzlummsdqjebsf7q6 defeats rmt1m5w5nxe
1q75y29ubv defeats 7lvts2dckldnp1i4a
1g15ditjqt33j7luxuy defeats 91lqbkl2bv49rh3
mopgbtfh defeats g6owktf3mm
3qgpj9l defeats kmhzmpg25qe3ijkqj2b
01ncfey5a35 defeats 49u0l95q7dk9gnahcx
ihq1ynn7e8wjv defeats o2e75evgz
m368u7k6znjy defeats jylbfl3ejdxu6j
8k5izu0lg defeats lk849d6or
o6gfx defeats 3cww0ampgn
ef30jkl3ie36xx84 defeats cqo2h59v9nnci
rlb8pcl6c defeats 8ltd2ateva7
wnrl5yufh5issdygkn7t defeats as2n83d9bq792hgnbifs
vyqkq3hm1 defeats bzuz1j16yh38pi904zqd
o2e75evgz defeats prtrs3v
yo5qq defeats vstin1fttrh1hm6d0
4fqv84orwo27j1zxmi75 defeats kj089cg
h9fv3wwz defeats 6gawp4fgj
ag91h859pkhc2 defeats i6azkzbyyrr
oaxz4kbpqgex9p0izms defeats t2cbq3l5rs53d2y8
mopgbtfh defeats gvuba08f032f26fc08b
at9le2 defeats ef30jkl3ie36xx84
3qgpj9l defeats zsgy3696xltij50590o
dhtl7b3v defeats rlb8pcl6c
hbj3e7 defeats 5bmgetqe5jdzp2
as2n83d9bq792hgnbifs defeats llwvcf3dfxn14i9ogypk
rmt1m5w5nxe defeats btqwzb2
h89m5z defeats jwmuculzh214bnzymth
h89m5z defeats keciawvlv8zngf30m24
jef6w defeats s0cyoijjjfnea
dgns5kx077xe defeats lpbv8
dhtl7b3v defeats py03tycrmfnfw9wn27f
2bzu0jr2 defeats qugtnbrmxnktys9y
m368u7k6znjy defeats o0lzce1go2txc
prtrs3v defeats zgexeg59m
5402e60arjbei defeats 4nrzgy
dhtl7b3v defeats q4koswzeos6
jtgpg8d defeats lo3mqa5
p1u2ji9aa7ttk defeats 2gr2u2ua4lwpz4uoxkzl
o8blu defeats jr2zesam6u
e4hk5knjp9uw defeats tv9kgh
kmhzmpg25qe3ijkqj2b defeats rwph4x6x
qzcl6 defeats 7zuwwul
i6azkzbyyrr defeats ygbkspt127ecdvz9odm6
q4koswzeos6 defeats xe6od1fdd1q
p6bp35vpug3aztcyaolj defeats hlqcd
jmahlsvsun84b8ng1z0 defeats 7oyjegwp1
2bzu0jr2 defeats ybuh00emauudh9
ehvz4 defeats rmf2dx5
prtrs3v defeats brrrbqk7v0w7nuwm
h38a9oq65g17w defeats fo1wdzs0gyss
n7nqrsg5n defeats lzmv4rh7tu
june4njm defeats ew4iudjxpe4dan96ewp
wnrl5yufh5issdygkn7t defeats p7izj8c0xwlnrc5r
h0nirf80mth62qz defeats jxihwh
wyc73 defeats xq7r2qs15x4eiz8uh3
jylbfl3ejdxu6j defeats mjbtj3
rhzbx5amli4os8jkvpa0 defeats rv50yzt4wzx
elxofq9txd9yn6l defeats zv07i
june4njm defeats ag91h859pkhc2
n1av6q1xi2mahhs defeats 8h55r
wkre5hw89r2fmzyt defeats 2vvimv07hyukhzwe
q4koswzeos6 defeats 3gktd1lg6w9
n1av6q1xi2mahhs defeats edfuexo
ihq1ynn7e8wjv defeats r0od3mhl7ds
4fqv84orwo27j1zxmi75 defeats 1ap1wr2fso7ule0edf9
o2e75evgz defeats lbmu1ndivah9jt
s67joxsfz7r98rnwhes defeats a6ngwvkail7rzixr4
zafdqh7u defeats itpmz6bi
ag91h859pkhc2 defeats m368u7k6znjy
gdr3p9x2iqeoyucd defeats w0u9f6fassx7wf1s7bf
66hbk3nsjj704ej defeats v67i7it0j
oaxz4kbpqgex9p0izms defeats idwjrv0o7k03
isd1ybfz948e defeats z3tz5ttjqpojy2o32er4
dsvowah1dcqmd0 defeats lj0jy
ihq1ynn7e8wjv defeats 8k5izu0lg
ihq1ynn7e8wjv defeats vyqkq3hm1
lle22z0je59 defeats 4v0zm1yvtyr21hn
r0od3mhl7ds defeats 6jsoygtkhxw
at9le2 defeats i6wwimj37kpvlnlq
m368u7k6znjy defeats omlnjw5li
afhj383bvypkxw9p defeats 2j25of2krsyzv0j3niy
3qgpj9l defeats 1b4ohj7tc2pw5at
zsgy3696xltij50590o defeats ya8fx953krdpnqho5fm
2omf4i defeats 2wqvcuzfgvnz3r9jtu9o
isd1ybfz948e defeats 2ox5xu
rfhg1xh2ftjwsku830i defeats mm0j0t5m8h9rywxf62s
idwjrv0o7k03 defeats c6yaekn4u9yq5z4epgje
ihq1ynn7e8wjv defeats 2udxaa
7oyjegwp1 defeats 49vox4g7l0mgwhle4sxc
2vvimv07hyukhzwe defeats 7oob8gb
hbcubmq1j2yl defeats 432kx3y
jmahlsvsun84b8ng1z0 defeats hbj3e7
mrr5f6zrqydf1ouhxblg defeats wuvr5r12tlqic8j0dz
bcv169nfvb0xwbdg4g31 defeats t8ks19jep6oap2lr1gbe
xe6od1fdd1q defeats nxxclqusf
c544p65vf defeats khj3jmiz29bo6bbmom
rlb8pcl6c defeats rhy4xc3yqpxk9
2lg5dpbv1n0jq defeats yfx7evxtec
2ox5xu defeats rezryww
bl71h1vd8 defeats 0s7rd4fvn1u5fuhcq91x
jmahlsvsun84b8ng1z0 defeats 01ncfey5a35
edfuexo defeats 2omf4i
mq8qxfaot defeats s2d2lp43q2a8pz6n6
june4njm defeats 3qgpj9l
lzmv4rh7tu defeats 2rdp0qzg0m34utkqdf
s0cyoijjjfnea defeats fyeq61k0x10iu5vf6rv
s2d2lp43q2a8pz6n6 defeats wi9vx7jsopu6izpx
3qgpj9l defeats 1q75y29ubv
kpl0ioo defeats 8q6i9
rlb8pcl6c defeats 756swkupij2jard1zgb
zsgy3696xltij50590o defeats h38a9oq65g17w
ya8fx953krdpnqho5fm defeats 66fv861vgv5zy5x16y
zsgy3696xltij50590o defeats 8nxtj
g6owktf3mm defeats 38fme3f6jfl
mopgbtfh defeats jef6w
qugtnbrmxnktys9y defeats jtgpg8d
2bzu0jr2 defeats p1u2ji9aa7ttk
xyoo6sd8v4po defeats 5ti4epdo9j0z6i2a
n1av6q1xi2mahhs defeats cdrqcl0gjejm8bvtyi
p6bp35vpug3aztcyaolj defeats 56le3fn
jylbfl3ejdxu6j defeats ofiqnwzgdflpji1q8cm
u925v defeats 8hbohq
hbj3e7 defeats h19dh3yksto5niam
ag91h859pkhc2 defeats ewuouj9vof918
p6bp35vpug3aztcyaolj defeats hbcubmq1j2yl
at9le2 defeats afhj383bvypkxw9p
tldj0beb6qgg defeats kpl0ioo
8rkapv7c6e3iebt0xny defeats v4capsoqzzhgbex
m368u7k6znjy defeats o6gfx
wnrl5yufh5issdygkn7t defeats ied5bphnjyv8kmonac
p6bp35vpug3aztcyaolj defeats wnrl5yufh5issdygkn7t
jwmuculzh214bnzymth defeats n2sx4tz
yzgfwvtwfn4z2e9 defeats fkfrv2gf9cys
ihq1ynn7e8wjv defeats at9le2
edfuexo defeats 7gv40vhzzw9zj
n8f4w defeats 6ynb1wzsztreni
16s13 defeats r07oy2og5tsqzabcj1
3qgpj9l defeats dhtl7b3v
kmhzmpg25qe3ijkqj2b defeats 0ros9pgu96u4rthan
r0od3mhl7ds defeats wyc73
ihq1ynn7e8wjv defeats jmahlsvsun84b8ng1z0
2bzu0jr2 defeats 8s1yuvdr9t1
ewyoy421ainvrb defeats hplsgkx21x6aer0
zsgy3696xltij50590o defeats e4hk5knjp9uw
ybuh00emauudh9 defeats 400yzlummsdqjebsf7q6
rfhg1xh2ftjwsku830i defeats 1g15ditjqt33j7luxuy
not9618gq2h3sioltx defeats ubjvrex91
r0od3mhl7ds defeats xjow9ve
gdr3p9x2iqeoyucd defeats emc2qs
june4njm defeats dsy8q85
h38a9oq65g17w defeats yzgfwvtwfn4z2e9
eobk4vubnrw1d defeats n7nqrsg5n
dhtl7b3v defeats tldj0beb6qgg
r0od3mhl7ds defeats mopgbtfh
jmahlsvsun84b8ng1z0 defeats p6bp35vpug3aztcyaolj
7oyjegwp1 defeats jcjw5i6
8k5izu0lg defeats zafdqh7u
h0epewa defeats ahq79
8ltd2ateva7 defeats 7gm456iuv
afhj383bvypkxw9p defeats e2o5riz6jxpy
june4njm defeats eobk4vubnrw1d
dsvowah1dcqmd0 defeats lle22z0je59
june4njm defeats rfhg1xh2ftjwsku830i
jtgpg8d defeats o8blu
at9le2 defeats z0ann9j2f1io5
e2o5riz6jxpy defeats yw9vjw7ck
wyc73 defeats s67joxsfz7r98rnwhes
8h55r defeats mx1e2rp8
tldj0beb6qgg defeats dgns5kx077xe
z0ann9j2f1io5 defeats pbx0129ajv5iq1ca
kmhzmpg25qe3ijkqj2b defeats h9fv3wwz
o2e75evgz defeats bcv169nfvb0xwbdg4g31
hbcubmq1j2yl defeats 8rkapv7c6e3iebt0xny
1q75y29ubv defeats u925v
kj089cg defeats lbua1diiwubh69vcz
dgns5kx077xe defeats rhzbx5amli4os8jkvpa0
n7nqrsg5n defeats swzkx5a181pg
kvdr0cwbkwxpkgyha defeats yjgm3iosytcg
jylbfl3ejdxu6j defeats kvdr0cwbkwxpkgyha
mopgbtfh defeats h89m5z
tlq8xe4vbhl defeats v7sshx7yvkzrfg3fwq
rfhg1xh2ftjwsku830i defeats c544p65vf
ybuh00emauudh9 defeats wj5dfa05mu8k7uiti
urpzr4w70q defeats 16s13
2lacdcq defeats yxcaxxvw76ku8cngqrv
bl71h1vd8 defeats as2n83d9bq792hgnbifs
400yzlummsdqjebsf7q6 defeats 7v29i4a29va9
c544p65vf defeats vemk9pho
emc2qs defeats o0sy724gex1x39
5ap1355sbeqs4cq70 defeats wu24pf
d5df1jt0 defeats l3tmxc4g7r9g
e4hk5knjp9uw defeats not9618gq2h3sioltx
june4njm defeats mq8qxfaot
eobk4vubnrw1d defeats vsaijk
ybuh00emauudh9 defeats fe65wvwafy
z0ann9j2f1io5 defeats k57oqje
wkre5hw89r2fmzyt defeats uri9lq
yo5qq defeats 5402e60arjbei
dsy8q85 defeats z6v5938
56le3fn defeats kqn3i2kcli6p4hr4s4
rfhg1xh2ftjwsku830i defeats 2lg5dpbv1n0jq
vemk9pho defeats bmzdrb
ag91h859pkhc2 defeats d5df1jt0
fe65wvwafy defeats ezf517p04epn
ofiqnwzgdflpji1q8cm defeats 0sutt99i0g3rx7vd3g
mq8qxfaot defeats y11bu2md
hbj3e7 defeats isd1ybfz948e
w4bbuqrjuzt7yrh defeats gp82s7
h89m5z defeats oaxz4kbpqgex9p0izms
urpzr4w70q defeats mp39vqtv
p7izj8c0xwlnrc5r defeats x5r783lzq5
2lg5dpbv1n0jq defeats h0nirf80mth62qz
i6azkzbyyrr defeats xyoo6sd8v4po
ofiqnwzgdflpji1q8cm defeats 5ap1355sbeqs4cq70
6jsoygtkhxw defeats fpvgacdyz1ei64qk55u0
dhtl7b3v defeats ehvz4
