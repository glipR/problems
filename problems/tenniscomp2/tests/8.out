bl71h1vd8 must be the cheater
They cheated on 1 games
bl71h1vd8 defeats as2n83d9bq792hgnbifs
